source/index.html

```diff
@@ -1,39 +1,282 @@
+                        <ul>
+                            <li>Who have high energy</li>
+                            <li>Who are lonely</li>
+                            <li>Who have separation anxiety</li>
+                            <li>Who are destructive when home alone</li>
+                            <li>Who need special attention</li>
+                            <li>Who need some confidence building</li>
+                            <li>Or anyone else who needs to release some energy, be social, and have fun.</li>
+                        </ul>
+                        <h3>Requirements:</h3>
+                        <ul>
+                            <li>2 hour behavior assesment for first visit</li>
+                            <li>Current rabies, distemper and bordetella vacination</li>
+                            <li>Completed Daycare enrollment form. <a href="./assets/daycare_form.pdf" download="daycare form"><em>download here</em></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <aside>
+                    <img src="./assets/cocker.png">
+                    <img src="./assets/beach.png">
+                    <br>
+                    <img src="./assets/dogs.png">
+                    <img src="./assets/dog-11.png">
+                    <br>
+                    <img src="./assets/dog-16.png">
+                    <img src="./assets/mini.png">
+                    <br>
+                    <img src="./assets/waterfall.png">
+                    <img src="./assets/dog-6.png">
+                </aside>
+            </div>
+        </section>
+        <section id="boarding">
+            <img class="dog-divider" src="./assets/new-dog4.png">
+            <hr>
+            <div class="content">
+                <h2>BOARDING</h2>
+                <p>Marquette's first 24 hour staffed pet boarding facility with cage free options.</p>
+                <p>Dogs will be required to be vaccinated for Rabies, Distemper and Bordetella. Vaccinations must be adminstered no sooner than one week prior to boarding date. <em>- A titer antibody test will also be admissible with written consent from vet.</em></p>
+                <p>Completion of daycare enrollment form is required <a href="./assets/daycare_form.pdf" download="daycare form"><em>download here</em></a></p>
+                <div class="list-info">
+                    <h3>Sleeping Options:</h3>
+                    <ul>
+                        <li>4x5 Private Suite - $21 per night</li>
+                        <li>4x6 Comfort Condo - $24 per night</li>
+                        <li>4x7 suite <em>(includes A/C and soft rubber flooring)</em> - $27 per night</li>
+                        <li>4x8 Glass front condo - $27 per night</li>
+                        <li>8x10 Family suite <em>(private luxury suite with A/C and toddler beds)</em> - $42 per night for two dogs, $16 for each additional dog.</li>
+                    </ul>
```

Comment:
"Sentence case" I think here too. For example, `<li>4x5 Private suite - ` and `<li>4x7 Suite <em> ` etc.